kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/HighAvailabilityConf.scala

```diff
@@ -99,4 +99,11 @@ object HighAvailabilityConf {
     .timeConf
     .checkValue(_ > 0, "Must be positive")
     .createWithDefault(Duration.ofSeconds(120).toMillis)
+
+  val HA_ZK_ENGINE_LOCK_TIMEOUT: ConfigEntry[Long] =
+    buildConf("ha.zookeeper.engine.lock.timeout")
+      .doc("The timeout(s) of a lock when a engine created")
+      .version("1.0.0")
+      .timeConf
+      .createWithDefault(Duration.ofSeconds(30).toMillis)
```

Comment:
The default timeout number I think should be same with `session.engine.initialize.timeout` which is the max timeout before fail.